pkg/apis/migration/v1alpha1/directvolumemigrationprogress_types.go

```diff
@@ -27,20 +27,58 @@ import (
 
 // DirectVolumeMigrationProgressSpec defines the desired state of DirectVolumeMigrationProgress
 type DirectVolumeMigrationProgressSpec struct {
-	ClusterRef *kapi.ObjectReference `json:"clusterRef,omitempty"`
-	PodRef     *kapi.ObjectReference `json:"podRef,omitempty"`
+	ClusterRef  *kapi.ObjectReference `json:"clusterRef,omitempty"`
+	PodRef      *kapi.ObjectReference `json:"podRef,omitempty"`
+	PodSelector map[string]string     `json:"podSelector,omitempty"`
```

Comment:
I think you will need a `Namespace` field here right? in order to list using this selector?

Reply:
fixed